micropub.php

```diff
@@ -481,6 +496,37 @@ private static function handle_authorize_error( $code, $msg ) {
 		static::respond( $code, $msg );
 	}
 
+	private static function post_status( $mf2 ) {
+		$props = $mf2['properties'];
+		if ( isset( $props['visibility'] ) ) {
+			if ( array( 'private' ) === $props['visibility'] ) {
+				return 'private';
+			}
+			// Public or Private are the only options
+			if ( array( 'public' ) !== $props['visiblity'] ) {
+				return null;
```

Comment:
this should be an error too.

Reply:
Returning null causes the 400 error to be triggered.

Reply:
ah. great! that's worth a comment. also `visiblity` above is misspelled, which means we need a unit test for this error handling code path. :P

Reply:
ping!

Reply:
ping on commenting on each `return null` in this function that they result in an HTTP 400.

Reply:
thanks for the fixes! this request keeps falling through the cracks though. once these two comments are in, i'll happily merge!